### PR TITLE
Add headlines-dash_string

### PR DESCRIPTION
**Pull request by glapa-grossklag**
Allow users to specify the string that's repeated to display horizontal dashes.

### CODE REVIEW

File: doc/headlines.txt [hunk 1]
```diff
@@ -165,6 +174,9 @@ fat_headlines                                        *headlines-fat_headlines*
 ==============================================================================
  4. CHANGELOG                                            *headlines-changelog*
 
+2.1.2
```
Comment (lukas-reineke) [inline doc/headlines.txt:177]: should be 2.2.0
new feature means minor patch. But you can remove this here, I'll make a new commit to update the version when I merge it into master.

**Review by lukas-reineke — CHANGES_REQUESTED** [review]
thank you, looks good
but can you rebase your branch against develop please

**Comment by glapa-grossklag** [comment]
I've rebased against `develop` and removed the version -- let me know if this looks good.

**Comment by lukas-reineke** [comment]
you did something wrong with the rebase, you also rewrote commits from develop.
Just check out your branch and run `git rebase develop` again, then force push.

**Comment by glapa-grossklag** [comment]
Hopefully that's better. If it's not, I can just open up a new PR properly on the develop branch.

**Review by lukas-reineke — APPROVED** [review]
okay now, thank you :rocket:

**Pull request merged**